src/assets/styles/styles/custom/_tables.scss

```diff
@@ -83,7 +103,13 @@
     }
 
     .selection {
-      background-color: $selection-color !important;
+      background-color: white !important;
+      outline-width: 1px;
+      outline-style: solid;
+      outline-color: white;
+      box-shadow: 0px 4px 7px rgba(16, 32, 70, 0.2), 0px 0px 7px rgba(16, 32, 70, 0.2);
+      overflow: hidden;
```

Comment:
`overflow: hidden` dodałam, żeby widzieć równą siatkę podczas pracy nad dropdownem, ale ogólnie to jest złe rozwiązanie, bo ucina "wystające" fragmenty prawej strony komórek
![picture](https://user-images.githubusercontent.com/48785655/107128222-59859880-68bc-11eb-86b0-fe79179ad4be.png)